frontend/src/app/app-routing.module.ts

```diff
@@ -6,13 +6,15 @@ import {AppComponent} from './app.component';
 import {AlreadyLoggedGuardService} from './shared/auth/guards/already-logged-guard.service';
 import {MySubmissionsComponent} from './user/my-submissions/my-submissions.component';
 import {AuthGuardService} from './shared/auth/guards/auth-guard.service';
+import {ConferenceDetailComponent} from './conference/conference-detail/conference-detail.component';
 
 
 const routes: Routes = [
   {path: '', component: AppComponent, pathMatch: 'full'},
   {path: 'signup', component: SignUpComponent, canActivate: [AlreadyLoggedGuardService]},
   {path: 'login', component: LogInComponent, canActivate: [AlreadyLoggedGuardService]},
-  {path: 'my-submissions', component: MySubmissionsComponent, canActivate: [AuthGuardService]}
+  {path: 'my-submissions', component: MySubmissionsComponent, canActivate: [AuthGuardService]},
+  {path: 'conferences/:name', component: ConferenceDetailComponent},
```

Comment:
i dont think using the name in the url makes much sense, change it to id (add id to conference dto) and fetch the conference (mocked) from conference service by this id

Reply:
also it should be conference, not conferences